code/modules/mob/living/basic/icemoon/ice_whelp/ice_whelp.dm

```diff
@@ -0,0 +1,82 @@
+/mob/living/basic/mining/ice_whelp
+	name = "ice whelp"
+	desc = "The offspring of an ice drake, weak in comparison but still terrifying."
+	icon = 'icons/mob/simple/icemoon/icemoon_monsters.dmi'
+	icon_state = "ice_whelp"
+	icon_living = "ice_whelp"
+	icon_dead = "ice_whelp_dead"
+	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	mouse_opacity = MOUSE_OPACITY_ICON
+	butcher_results = list(/obj/item/stack/ore/diamond = 3, /obj/item/stack/sheet/sinew = 2, /obj/item/stack/sheet/bone = 10, /obj/item/stack/sheet/animalhide/ashdrake = 1)
```

Comment:
```suggestion
	butcher_results = list(
		/obj/item/stack/ore/diamond = 3,
		/obj/item/stack/sheet/sinew = 2,
		/obj/item/stack/sheet/bone = 10,
		/obj/item/stack/sheet/animalhide/ashdrake = 1,
	)
```

multiline this list like so. can you install an extension for VSC to alphabetize as well? thanks